java/drivers/driver-hazelcast4plus/src/main/java/com/hazelcast/simulator/tests/cp/CPMapTest.java

```diff
@@ -73,58 +103,119 @@ private String[] createCpGroupNames() {
     public void prepare() {
         for (CPMap<Integer, String> mapReference : mapReferences) {
             for (int key = 0; key < keys; key++) {
-                mapReference.set(key, v);
+                mapReference.set(key, INITIAL_VALUE);
```

Comment:
Pick an item from 'v'.

Reply:
The reason why `INITIAL_VALUE` value is used here is to see whether it was really updated during the test. For that case it makes no sense to set it to any of expected values.

Reply:
Why don't you use null as initial value?

Reply:
CP Map currently doesn't allow `null` values. That's the reason why I added this `INITIAL_VALUE` setup.

Reply:
I meant: leave the CPMap empty when starting the test. So there is no data to insert.